src/components/Runner.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+<!--  Fix access to apiEndpoint-->
+  <v-list-tile avatar :href="'https://ci.linagora.com/api/v4/runners/' + item.id" target="_blank">
+    <v-list-tile-content>
+      <v-list-tile-title id="name">
+        <span>{{ item.name }}</span>
+      </v-list-tile-title>
+      <v-list-tile-sub-title>
+        <span v-bind:class="statusColor" >{{ item.status }}</span> - {{ item.description }}
+      </v-list-tile-sub-title>
+    </v-list-tile-content>
+
+    <v-list-tile-action v-if="!hideActions">
+      <div id="actions">
+      </div>
+    </v-list-tile-action>
+  </v-list-tile>
+</template>
+
+<script>
+
+export default {
+  name: "Runner",
+  props: {
+    item: Object,
+    hideActions: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    statusColor: function () {
+      return {
+        'status-green': this.item.status === 'online',
+        'status-red': this.item.status === 'offline',
+        'status-yellow': this.item.status === 'paused'
+      }
```

Comment:
Look mama I am doing JS :)

```js
return STATUS_COLORS[this.item.status];
```

And first create a constant before export:

```js
const STATUS_COLORS = {
  online: "status-green",
  offline: "status-red",
  paused: "status-yellow
};
```

Reply:
Easy ;)